src/Symfony/Bundle/DebugBundle/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+7.4
+---
+
+ * Wire the `$profilerDumper` argument in `DumpListener`
```

Comment:
what's the outcome of this change? is this related to the console profiling? if not, is that a bugfix for another branch? Sorry I forgot how this works :)

Reply:
@nicolas-grekas It wires the newly added second dumper argument so that the `DumpListener` can "choose" which dumper to use based on whether the `--profile` option is set. The listener is only triggered for console commands. Like I mentioned earlier, it feels more like a missing feature to me than a bug fix.